src/window/mod.rs

```diff
@@ -169,6 +169,7 @@ impl<F: Frame + 'static> Window<F> {
     ///
     /// Providing non `None` value for `theme_manager` should prevent theming pointer
     /// over the `surface`.
+    #[allow(clippy::match_like_matches_macro)]
```

Comment:
Also please don't add unrelated changes like this. Even if the clippy builds fails, it is unrelated to your changes so you don't have to worry about it.

This should be done in an other PR, and either by fixing the code to appease the lint, or by allowing the lint at the crate level if fixing the code would require bumping the minimal rust version.